OneCommunityGlobal/HighestGoodNetworkApp
# Kaixiang - make bio announcement obvious

# Description
<img width="760" alt="Screenshot 2023-07-15 at 10 25 04 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/98206107/2a175dbe-210b-4606-bbd6-862f306678e8">


## Main changes explained:
Added a yellow background on bio announcement when the user meets two criteria. Total tangible hours is greater than 80 hours and has been on the team for at least 60 days.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to dashboard→ Reports→ Weekly Summary Report
5. Check if the yellow background display correctly if the user meets two criteria.
6. Toggle bio announcement to "Posted" and refresh the page, yellow bar should disappear.
7. Toggle back to "Neutral" or "Requested", yellow bar should come back.

## Screenshots or videos of changes:
<img width="1132" alt="conditional rendering UI" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/98206107/daeb18b9-bdbf-4ebf-a74d-04eebb5eae69">
